yb_migrate/cmd/export.go

```diff
@@ -224,15 +216,15 @@ func validateOracleParams() {
 	} else if source.TNSAlias != "" {
 		//Priority order for Oracle: oracle-tns-alias > source-db-name > oracle-db-sid
 		fmt.Println("Using TNS Alias for export.")
-		log.Infof("Using TNS Alias for export.")
+		log.Info("Using TNS Alias for export.")
```

Comment:
nit: consider using `utils.LogAndPrint(msg)` Also if you can modify if some better function name comes to your mind, maybe `InfoPrintf()`....